nexus/tests/integration_tests/schema.rs

```diff
@@ -456,29 +470,50 @@ const REFERENTIAL_CONSTRAINTS: [&'static str; 8] = [
 const VIEWS: [&'static str; 4] =
     ["table_catalog", "table_schema", "table_name", "view_definition"];
 
-const STATISTICS: [&'static str; 8] = [
+const STATISTICS: [&'static str; 11] = [
     "table_catalog",
     "table_schema",
     "table_name",
     "non_unique",
     "index_schema",
     "index_name",
+    "seq_in_index",
     "column_name",
     "direction",
+    "storing",
+    "implicit",
```

Comment:
"implicit" was the one we ended up using for the "column comparison" regression tests, but the others also seem useful to have.